temporian/implementation/numpy/data/display_utils.py

```diff
@@ -77,6 +77,9 @@ def display_html(evset: EventSet) -> str:
         index_data = evset.data[index_key]
         num_timestamps = len(index_data.timestamps)
         max_timestamps = config.display_max_events or num_timestamps
+        #Slices timestamps and features if there are more than 7 events
+        display_timestamps = np.concatenate((index_data.timestamps[:3], index_data.timestamps[-3:])) if num_timestamps > 7 else index_data.timestamps[:num_timestamps]
```

Comment:
The user can specify a `print_max_events` different than 7 in `temporian/utils/config.py`, so we need to use that constant instead of hardcoding the number here. Note that you'll also need to compute what half of that constant is (to compute the equivalent of `3` here - you'll need to round it down if it's an odd number)